src/amazons3/amazons3_client.bal

```diff
@@ -386,5 +386,5 @@ public type ClientConfiguration record {
     string accessKeyId;
     string secretAccessKey;
     string region = DEFAULT_REGION;
-    http:ClientConfiguration clientConfig;
+    http:ClientConfiguration clientConfig = {http1Settings: {chunking: http:CHUNKING_NEVER}};
 };
```

Comment:
Shall we reflect this change in the Module.md and the README.md also?

Reply:
@Nipunich will check this.

Reply:
@dulanjalidilmi fixed it in https://github.com/wso2-ballerina/module-amazons3/pull/23